CHANGELOG.md

```diff
@@ -1,11 +1,22 @@
+# v2.3.4 (2024-08-19)
+## OS Changes
+
+* Update libnvidia-container to v550.54.14 and nvidia-container-toolkit to v1.16.1 ([#88])
+
+## Settings API Changes
```

Comment:
Have we used this category before? The settings API is part of the OS, so I would just include it in "OS Changes".